sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -159,6 +159,17 @@ public boolean isElementVisibleWithTimeout(By selector, int seconds) {
     return true;
   }
 
+  public boolean isElementSelected(By selector) {
+    try {
+      assertHelpers.assertWithPoll(
```

Comment:
Please use `assertWithPoll15Second` if we need to change the time it will be easier, or the `FLUENT_WAIT_TIMEOUT_SECONDS`